src/sidebar/components/sidebar-view.js

```diff
@@ -122,8 +124,17 @@ function SidebarView({
       frameSync.focusAnnotations([linkedAnnotationAnchorTag]);
       frameSync.scrollToAnnotation(linkedAnnotationAnchorTag);
       selectTab(directLinkedTab);
+    } else if (linkedAnnotation) {
+      // Make sure to allow for orphaned annotations (which won't have an anchor)
+      selectTab(directLinkedTab);
     }
-  }, [directLinkedTab, frameSync, linkedAnnotationAnchorTag, selectTab]);
+  }, [
+    directLinkedTab,
+    frameSync,
+    linkedAnnotation,
```

Comment:
By adding `linkedAnnotation` to the dependencies here, tab selection will be re-evaluated on the change of `$orphan: true` on orphaned annotations. Otherwise this executes prematurely.